### DISCUSSION TITLE
Can't use text fonctionality

### DISCUSSION THREAD

**Discussion by aliounebfam** [tldraw/tldraw-v1#554] [Show and tell]
I can't add text...

**Comment by judicaelandria** [comment]
i think you can use it now

**Reply by aliounebfam** [reply]
No, i can't
When i try, he change automatically to "select" option

**Reply by judicaelandria** [reply]
https://user-images.githubusercontent.com/46365844/191406681-f8d5d8ff-2619-4e92-b8f2-7ee74a0b35c8.mov

**Reply by steveruizok** [reply]
Still an issue?
